diff --git a/src/main/java/org/pegdown/ToHtmlSerializer.java b/src/main/java/org/pegdown/ToHtmlSerializer.java
--- a/src/main/java/org/pegdown/ToHtmlSerializer.java
+++ b/src/main/java/org/pegdown/ToHtmlSerializer.java
@@ -456,7 +456,7 @@ public class ToHtmlSerializer implements Visitor {
         for (LinkRenderer.Attribute attr : rendering.attributes) {
             printAttribute(attr.name, attr.value);
         }
-        printer.print(" />");
+        printer.print("/>");
     }
 
     protected void printLink(LinkRenderer.Rendering rendering) {
